test/docker/index.js

```diff
@@ -1,27 +1,29 @@
+/*global describe*/
+
 'use strict';
 
-var os      = require('os');
-var proc    = require('child_process')
-var config  = require('../knexfile');
-var knex    = require('../../knex');
+const os      = require('os');
+const proc    = require('child_process')
+const config  = require('../knexfile');
+const knex    = require('../../knex');
 
 if (canRunDockerTests()) {
-  var dialectName;
-  for (dialectName in config) {
+  for (const dialectName in config) {
     if (config[dialectName].docker) {
-      require('./reconnect')(config[dialectName], knex);
+      describe(`${dialectName} dialect`, function() {
+        require('./reconnect')(config[dialectName], knex);
+      })
     }
   }
 }
 
 function canRunDockerTests() {
-  var isLinux   = os.platform() === 'linux';
-
+  const isLinux   = os.platform() === 'linux';
+  const isDarwin  = os.platform() === 'darwin'
   // dont even try on windows / osx for now
-  var hasDocker = false;
-  if (isLinux) {
+  let hasDocker = false;
+  if (isLinux || isDarwin) {
```

Comment:
I cant remember why I didn't enable this to osx earlier, I'm pretty sure there was a reason for that, but this is fine like this. If it causes problems it can always be changed.

Reply:
I didn't find any issues. Maybe the problem before was that docker machine and docker for mac did not support the unix domain socket at `/var/run/docker.sock`... It has been supported for some time now, so I think this should be ok.

Reply:
Yeah I think that was the reason 👍